src/components/Settings/SettingsJobsCache/index.tsx

```diff
@@ -51,6 +62,13 @@ const messages: { [messageName: string]: MessageDescriptor } = defineMessages({
   'sonarr-scan': 'Sonarr Scan',
   'download-sync': 'Download Sync',
   'download-sync-reset': 'Download Sync Reset',
+  editJobSchedule: 'Modify Job Schedule',
+  editJobScheduleDescription:
+    'A job schedule can be represented by a CRON expression. Use a tool like <cronhub>crontab.cronhub.io</cronhub> to generate valid CRON expressions.',
+  jobScheduleEditSaved: 'Job schedule modified successfully!',
+  jobScheduleEditFailed: 'Something went wrong while saving the job schedule',
+  editJobSchedulePrompt: 'Enter CRON expression',
+  validationJobSchedule: 'You must provide a CRON expression',
```

Comment:
I thought we weren't going to allow users to enter the cron expression directly?

Reply:
Yes, this was just what I was already working on. I'll change it to the presets next.